_posts/2021-08-03-community-clients.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title:  "Carrying forward the OpenSearch client libraries as a community"
+authors: 
+  - elifish
+date:   2021-08-03 01:01:01 -0700
+categories: 
+  - community
+---
+
+Software projects often provide language specific client libraries to make it easy to integrate applications. OpenSearch is built to be wire compatible with open source Elasticsearch 7.10.2 so that anyone can leverage the existing clients, connectors, and tools with minimal modifications to their deployments.
+
+Starting with version 7.14, multiple of the [clients maintained by Elastic](https://www.elastic.co/guide/en/elasticsearch/client/index.html) contain [new logic](https://github.com/elastic/elasticsearch-py/pull/1623) that rejects connections to OpenSearch clusters or to clusters running open source distributions of Elasticsearch 7. This includes the [Apache-licensed distribution](https://www.elastic.co/downloads/past-releases/elasticsearch-oss-7-10-2) provided by Elastic, as well as community distributions like [Open Distro for Elasticsearch](https://opendistro.github.io/for-elasticsearch/).
+
+For the time being, users of these client libraries should not upgrade to version 7.14 as this may break applications. Please see these instructions for ensuring you’re using client connector libraries that work with open source Elasticsearch and OpenSearch clusters.
```

Comment:
"instructions" should link to docs. @aetter is there a PR for client compatibility we should reference here?

Reply:
found the link: https://opensearch.org/docs/clients/index/